tools/mage/dev.go

```diff
@@ -155,6 +239,15 @@ func (Dev) InitStack() error {
 	)
 }
 
+// StartDevStack starts TTS in end-to-end test configuration.
+func (Dev) StartDevStack() error {
+	os.Setenv("TTN_LW_IS_DATABASE_URI", databaseURI)
```

Comment:
I would expect a running DB to be a dependency of this.
If the reason why it's not is that DB may be started externally (i.e. not via this mage file), how about a target, which would check if DB is available, and, if not, starts the DB?